docs: typos and grammar fixes (#180)
nuxt/image: docs/content/en/components/nuxt-img.md

---
title: Usage of <nuxt-img> component
description: 'Discover how to use and configure the nuxt-img component.'
menuTitle: <nuxt-img>
category: Components
position: 201
---

`<nuxt-img>` is a replacement for the native `<img>` tag with no additional styles and supporting all native attributes.

> - Use built-in static provider to optimize local and remote images

Reviewer: ```suggestion
- Uses built-in static provider to optimize local and remote images
```

- Converts `src` to provider URL

> - Automatically resize images based on `width` and `height` attributes

Reviewer: ```suggestion
- Automatically resizes images based on `width` and `height` attributes
```

> - Support `loading="lazy"` with wider range of browsers (see [native support](https://caniuse.com/loading-lazy-attr))

Reviewer: ```suggestion
- Supports `loading="lazy"` with wider range of browsers (see [native support](https://caniuse.com/loading-lazy-attr))
```

> - Automatically generate `alt` attribute based on `src`.

Reviewer: ```suggestion
- Automatically generates `alt` attribute based on `src`.
```

- This value will set as `width` attribute of DOM element
- On `layout != "responsive"`
  - When both `width` and `height` are present and values aren't equal to `auto`, this value will be used to resize the image.
  - On server-side rendering when the value equals `auto` and height is not set or equal to `auto`, `nuxt-img` will automatically detect the image's size and use it. (This does not happens on client-side rendering)
  - On server-side rendering when the value equals `auto` and `height` is set, `nuxt-img` will automatically detect the image's size and calculate the proper width based on the image's aspect ratio and `height` value. (This does not happens on client-side rendering)

>   - On server-side rendering the image will resize using the calculated `width`

Reviewer: ```suggestion
  - On server-side rendering the image will be resized using the calculated `width`
```

> - This value will set as `height` attribute of DOM element

Reviewer: ```suggestion
- This value will be set as `height` attribute of DOM element
```

The `fit` property specifies the size of the images.
There are five standard values you can use with this property.

> - `cover`: (default) Preserving aspect ratio, ensures the image covers both provided dimensions by cropping/clipping to fit

Reviewer: ```suggestion
- `cover`: (default) Preserving aspect ratio, ensure the image covers both provided dimensions by cropping/clipping to fit
```
